addresses/frontend-mainnet.json

```diff
@@ -5,11 +5,15 @@
         "trancheFactoryAddress": "0x62F161BF3692E4015BefB05A03a94A40f520d1c0",
         "userProxyContractAddress": "0xEe4e158c03A10CBc8242350d74510779A364581C",
         "weightedPoolFactoryAddress": "0x8E9aa87E45e92bad84D5F8DD1bff34Fb92637dE9",
+        "stecrvAddress": "0x06325440d014e39736583c165c2963ba99faf14e",
```

Comment:
Minor, the original capitalization is: `0x06325440D014e39736583c165C2963BA99fAf14E`

Reply:
```suggestion
        "stecrvAddress": "0x06325440D014e39736583c165C2963BA99fAf14E",
```